### DISCUSSION TITLE
I am developing an android app using Kotlin. Is it possible to embed slit ui within android activity?

### DISCUSSION THREAD

**Discussion by surya239** [slint-ui/slint#6818] [Q&A]

**Comment by ghost** [comment]
@ogoffart ?

**Comment by ogoffart** [comment]
At the moment, Rust is the only supported language on android. But we plan to extend to more languages. It is always possible to call java/kotlin code from rust using JNI.

What usecase do you have in mind?

**Reply by surya239** [reply]
I am trying to develop cross platform components using slint. We successfully embedded slint within qt as a widget.Is it possible to render slint widget in android surface view
